cdap-ui/app/services/helpers.js

```diff
@@ -221,6 +221,35 @@ angular.module(PKG.name+'.services')
 
   /* ----------------------------------------------------------------------- */
 
+  function validNamespaceResolver(stateParams, myNamespace) {
+    const { namespace } = stateParams;
+    myNamespace
+      .getList()
```

Comment:
this is fetching the namespace right? Can we use the NamespaceStore instead?

Reply:
fixed